src/Types.tsx

```diff
@@ -0,0 +1,39 @@
+export type ResponseSectionsResults = {
```

Comment:
I would suggest you moving this file and `custom.d.ts` file to `types` directory to clean up a bit files structure in the project.
Then I would move all components to `components` directory and them every component with related `.types` file to directory specific for this component.
And to clean up it even further I would move files containing utility functions or function for fetching data to directory like `lib` or `utils`.